### ISSUE TITLE
Finances

### ISSUE THREAD

**Issue by dbrgn** [SwissPy/swisspycon#8]
Decide how to handle finances.

**Comment by dbrgn** [comment]
Things that need to be covered:
- Venue
- Website / server
- Coffee / snacks
- Lunch
- Helpers / speakers
- Maybe venue cleaning staff
- Conference badges / lanyards / name tags
- Printed stuff

Anything else?

---

We can't make any profits due to rules of the venue.

My suggestions:
- Venue is sponsored by HSR
- Get a single room, do 1 track
- Collect an entrance fee of 5-10 CHF
- Find a sponsor for coffee / snacks
- Find a sponsor for website
- Maybe find a sponsor for the lunch, otherwise everyone pays his own food, it's cheap
- Helpers and speakers don't get paid, it's a voluntary community thing
- Create a very simple conference booklet, maybe 8 A5 pages (2 A4 pages folded) with things like the program, a map, local "tourist" information, sponsors, etc
- Come up with a cool conference badge, maybe something unconventional that contains electronics? E.g. a small PCB that runs snake.
- Money is used to pay for badge, booklet, other print stuff, other expenses

**Comment by dbrgn** [comment]
Btw, HSR asked that we would allow their students to attend for free. Sounds reasonable, right? (I'd set a limit of max 20 free attendees though, if we're allowed to do that.)

**Comment by chiesax** [comment]
Sounds very reasonable. I would just suggest that we could reserve 20 free places for them (first come, first served) until some date (i.e. for instance 1 month before).

**Comment by dbrgn** [comment]
Good point :)

**Comment by jakeret** [comment]
Could we come up we an estimation on the total costs? This would help me when contacting sponsors (#13 )
- Does the HSR provide some catering service? 
  - How much are the costs for two coffebreaks? 
  - How much for the lunches?
- Cost for printing (#16 )
  - Booklets
  - Posters
  - Badges

Anything else?

**Comment by dbrgn** [comment]
> Does the HSR provide some catering service?

Yes. Coffee breaks will be offered by the cafeteria and lunch will be available at the [mensa](http://hochschule-rapperswil.sv-group.ch/de.html).

I don't know about the coffee breaks, but the lunch costs around 12 CHF per person excluding drinks (probably around 15 CHF including drinks).

@sfkeller do you know more about the actual costs?

**Comment by dbrgn** [comment]
Comment by @href:

> > So what are actually the 'minimal' expenses we will have? If I understand it correctly, we will have some fees for posters printing, but what else do we really need?
> 
> I would say drinks should be something we have at the venue, like say water bottles? Must stay hydrated ;). Other than that I also don't see much in terms of expenses.
> 
> > With 10 CHF we should surely be covered. And yes, we could donate the rest to the PSF or Djangogirls or some other organization like that. (We actually have to, because we're not allowed to make profit.)
> 
> +1, though I'm also okay with 5 bucks, I don't see how we need more. But it might look too cheap, so there's that.

**Comment by dbrgn** [comment]
> I would say drinks should be something we have at the venue, like say water bottles? Must stay hydrated ;). Other than that I also don't see much in terms of expenses.

We'll get coffee, water and bakery stuff from the HSR cafeteria. We just need someone to stand at the tables and help with the catering :) It'll cost us a bit though (per coffee).

Let's wait until we know the actual costs. I'll know them in a few days.

Do we also want some kind of conference badge for the people? That would require us to get lanyards and print the badges.

A booklet is probably not necessary, posters with the talk schedule should be enough.

Lunches can be covered by everyone on his/her own. The mensa is cheap. Otherwise there's a Migros and many restaurants in the old town.

There might be some additional cost for the cleaning people. Not sure about that yet.

**Comment by href** [comment]
> Do we also want some kind of conference badge for the people? That would require us to get lanyards and print the badges.

I'm not sure we should, unless we need it in terms of authentication/registration. Other than that it seems to be somewhat of a vanity article.

**Comment by dbrgn** [comment]
Yeah, we can probably just count on people actually registering and paying for the entrance, instead of checking each person.

**Comment by ccdesales** [comment]
> Do we also want some kind of conference badge for the people? That would require us to get lanyards and print the badges.

We could use some cheap name tags and markers.

**Comment by dbrgn** [comment]
OK, here are the infos regarding the food costs:
- Coffee / Tea:  2.50 / cup
- Croissant: 1.40 / piece
- Bakery: 2.70 / person
- Water 1L: 5.50
- Orange juice 1L: 6.50
- Coca Cola 0.5L PET: 3.50

According to @sfkeller the cost at the postgres conference was 350 for 51 people. So around 7 per person. (This is assuming that we serve the food / drinks ourselves.)

We can let the catering know about the definitive number of people 3 days in advance.

I'm still waiting on the confirmation whether or not there are still remaining costs for the room.

**Comment by dbrgn** [comment]
Alright, the venue _would_ cost something, but we won't have to pay that if we fulfill their rules (not making profit and showing them all our expenses).

About the name tags, which were you thinking about? The cheapest version I found including some kind of lanyard is this:
- https://adri.ch/de/lanyard-rund-unbedruckt/1872/ (1.00 CHF)
- https://adri.ch/de/ausweishullen-vertikal-90x60-mm/1578/ (0.60 CHF)

I think we won't gain much from getting something even cheaper. We can then print "templates" with the conference logo and an empty field where people can put their name with a marker.

So the cost breakdown is like this:
- Around 7-8 CHF per person for the food & bakery
- 192 CHF for 120 name tags

How much do the promotional posters/flyers cost? @chiesax wrote something about 75-100 CHF in #16. There will probably be some additional costs for the on-site posters. But we could probably print them on A3 at the HSR printer (or even plot them on A0 for around 4-8 bucks per poster).

**Comment by dbrgn** [comment]
@cdesales regarding potential sponsors, they could simply buy us 150 of these name tags. as long as we don't get money directly it's fine.

**Comment by ccdesales** [comment]
I was thinking more in name tags, which should be cheap to buy at office world or similar:

![2000px-hello_my_name_is_sticker svg](https://cloud.githubusercontent.com/assets/1451785/10849057/300b7506-7f1f-11e5-85f0-df6c72f9d3b0.png)

**Comment by dbrgn** [comment]
Would also work out. What do the others think? @href @chiesax @sfkeller @sfontana @sposs

**Comment by sposs** [comment]
Well, I think that for a first meeting, simple is better. No need for fancy, the cheapest the better.

**Comment by chiesax** [comment]
+1

**Comment by href** [comment]
+1

**Comment by sfontana** [comment]
+1

**Comment by dbrgn** [comment]
200 of these? :) http://www.amazon.de/Hello-Self-Adhesive-Name-Tags-451NT-42/dp/B001BYQ1HA/

**Comment by chiesax** [comment]
Good deal!

**Comment by dbrgn** [comment]
Can you order them through Suzie, @cdesales?

**Comment by ccdesales** [comment]
Will do.

**Comment by dbrgn** [comment]
OK, so if we spend around 7-8 CHF per person for the food & bakery and have no other expenses, we'd have 2-3 CHF per person for print stuff.

Regarding staff, I'd suggest getting colored lanyards with name tags for all helpers / organizers as well as for the speakers. Alternatively we could get sponsors for staff shirts.

**Comment by dbrgn** [comment]
OK, so if we spend around 7-8 CHF per person for the food & bakery and have no other expenses, we'd have 2-3 CHF per person for print stuff.

Regarding staff, I'd suggest getting colored lanyards with name tags for all helpers / organizers as well as for the speakers. Alternatively we could get sponsors for staff shirts.

Based on this information, I'd suggest fixing the entry price at 10 CHF, but making clear on the registration form that snacks but not lunch is included.

**Comment by jakeret** [comment]
Looks good. Nonetheless if we might have sufficient finance I might opt for a tag with the conference logo on it. (Maybe it's just me but I tend to keep the tags from nice conference that I have attended)

**Comment by href** [comment]
> Looks good. Nonetheless if we might have sufficient finance I might opt for a tag with the conference logo on it. (Maybe it's just me but I tend to keep the tags from nice conference that I have attended)

We should add nice-to-have things like this as options for sponsors.

**Comment by dbrgn** [comment]
Do we offer free entry to speakers? That would be another 80-100 CHF. I'd definitely support it though.

Also, we should probably note on the CfP page that we cannot reimburse any travelling costs.

**Comment by ccdesales** [comment]
I would say we should offer free entry to speakers.

Let's also not forget that one of the conditions for the HSR to lend us the venue is that some (30?) students get free access to the conference. If they're also going to consume snacks and coffee somebody's got to pay for that.

**Comment by chiesax** [comment]
Suggestion: why not have food and drinks as a surcharge. Entry ticket 5chf. Entry ticket and food 10chf. Student plus food 5chf. Then we draw or stick something on the tags. We assume people behave well and dont consume if not allowed.

**Comment by ccdesales** [comment]
I agree, we could add a surcharge for that.

**Comment by href** [comment]
Let's just keep it at 10 CHF, otherwise it gets too complicated in my opinion. The students which eat anyway can be paid by sponsoring. My company is willing to help with the costs there.

**Comment by jakeret** [comment]
I agree with Denis. Keep it simple and stupid

**Comment by sfkeller** [comment]
I don't expect 30 students from HSR. IMHO around 10 seems to be more realistic.

**Comment by dbrgn** [comment]
> Let's just keep it at 10 CHF, otherwise it gets too complicated in my opinion.

Agreed! I'd also expect around 10 students.

**Comment by dbrgn** [comment]
Oh, and I'd suggest that helpers can choose whether or not to pay. I'll gladly pay.

**Comment by dbrgn** [comment]
From meeting notes: Create a realistic budget.

**Comment by dbrgn** [comment]
See #64 for budget details.
